# 2.0.9 failed to compile on a system other than Win32/Linux/FreeBSD, this patch fixes it. Also fixes compile under GTK 2.0.x (old!)
--- src/common/util.c	2004-05-29 20:19:23.000000000 +1000
+++ src/common/util.c	2004-06-10 17:16:13.202029848 +1000
@@ -580,8 +580,8 @@
 {
 #if defined (USING_LINUX) || defined (USING_FREEBSD)
 	double mhz;
-	int cpus = 1;
 #endif
+	int cpus = 1;
 	struct utsname un;
 	static char *buf = NULL;
 
--- src/fe-gtk/fe-gtk.c	2004-05-17 01:42:33.000000000 +1000
+++ src/fe-gtk/fe-gtk.c	2004-06-07 23:23:15.000000000 +1000
@@ -725,8 +725,10 @@
 #if GTK_CHECK_VERSION(2,4,0)
 		if (gtk_window_is_active (GTK_WINDOW (sess->gui->window)))
 #else
+#if GTK_CHECK_VERSION(2,2,0)
 		if (GTK_WINDOW (sess->gui->window)->is_active)
 #endif
+#endif
 			return 1;	/* active/focused */
 
 		return 0;		/* normal (no keyboard focus or behind a window) */

